Formula/proteinortho.rb

```diff
@@ -0,0 +1,22 @@
+class Proteinortho < Formula
+  desc "Detecting orthologous genes within different species"
+  homepage "https://gitlab.com/paulklemm_PHD/proteinortho"
+  url "https://gitlab.com/paulklemm_PHD/proteinortho/-/archive/v6.0/proteinortho-v6.0.tar.gz"
+  sha256 "e82522cdae62c182fc3bb75903e8ce3a388573fccbea16b72a0a8c47a653a34b"
```

Comment:
```suggestion
  sha256 "e82522cdae62c182fc3bb75903e8ce3a388573fccbea16b72a0a8c47a653a34b"

```
Add a blank line after `sha256`.

Reply:
I added a blank in the next commit. The 9c18538, just fixes the file permission.